server/server.js

```diff
@@ -248,6 +249,10 @@ let needSetup = false;
         });
     }
 
+    if (process.env.NODE_ENV === "e2e") {
```

Comment:
Let's not add the new environment variable and just generally enable this in dev-mode.
This also cleans up that code a bit by not having a Middleware act as a router..

Reply:
Removed the middleware & added as routes inside the isDev block of server.